sequencer/synchronizer/synchronizer.go

```diff
@@ -155,19 +156,14 @@ func NewSynchronizer(
 		Rollup: *rollupConstants,
 	}
 
-	initVars, startBlockNum, err := getInitialVariables(ethClient /*&consts*/)
-	if err != nil {
-		return nil, common.Wrap(err)
-	}
-
+	startBlockNum := cfg.StartBlockNum
```

Comment:
@Intizar-T Do you think if it's better to add startBlockNumber manually via config? 🤔

Reply:
Yeah, hardcoded one is bad practice, but I don't think we need to reflect the `startBlockNum` in config